Compilation error after upgrade to prost v0.6.1

As of 01e91f5e8dcb88c236c1089b31c365f5ba997667:

```console
% cargo build
   Compiling wascc-actor v0.1.1 (/Users/gtb273/wrk/src/github.com/wascc/wascc-actor)
error[E0277]: the trait bound `wascc_codec::keyvalue::GetRequest: prost::Message` is not satisfied
  --> src/kv.rs:52:63
   |
52 |         host_call(&route(CAPID_KEYVALUE, OP_GET), &protobytes(cmd)?).map(|vec| {
   |                                                               ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::GetRequest`
   | 
  ::: src/lib.rs:59:8
   |
59 | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
   |        ----------           -------------- required by this bound in `protobytes`

error[E0599]: no function or associated item named `decode` found for type `wascc_codec::keyvalue::GetResponse` in the current scope
  --> src/kv.rs:53:37
   |
53 |             let resp = GetResponse::decode(vec.as_ref()).unwrap();
   |                                     ^^^^^^ function or associated item not found in `wascc_codec::keyvalue::GetResponse`
   |
   = help: items from traits can only be used if the trait is in scope
   = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
           `use prost::message::Message;`

error[E0277]: the trait bound `wascc_codec::keyvalue::SetRequest: prost::Message` is not satisfied
  --> src/kv.rs:68:63
   |
68 |         host_call(&route(CAPID_KEYVALUE, OP_SET), &protobytes(cmd)?).map(|_vec| ()).map_err(|e| e.into())
   |                                                               ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::SetRequest`
   | 
  ::: src/lib.rs:59:8
   |
59 | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
   |        ----------           -------------- required by this bound in `protobytes`

error[E0277]: the trait bound `wascc_codec::keyvalue::AddRequest: prost::Message` is not satisfied
  --> src/kv.rs:76:63
   |
76 |         host_call(&route(CAPID_KEYVALUE, OP_ADD), &protobytes(cmd)?).map(|vec| {
   |                                                               ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::AddRequest`
   | 
  ::: src/lib.rs:59:8
   |
59 | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
   |        ----------           -------------- required by this bound in `protobytes`

error[E0599]: no function or associated item named `decode` found for type `wascc_codec::keyvalue::AddResponse` in the current scope
  --> src/kv.rs:77:37
   |
77 |             let resp = AddResponse::decode(vec.as_ref()).unwrap();
   |                                     ^^^^^^ function or associated item not found in `wascc_codec::keyvalue::AddResponse`
   |
   = help: items from traits can only be used if the trait is in scope
   = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
           `use prost::message::Message;`

error[E0277]: the trait bound `wascc_codec::keyvalue::ListPushRequest: prost::Message` is not satisfied
  --> src/kv.rs:87:64
   |
87 |         host_call(&route(CAPID_KEYVALUE, OP_PUSH), &protobytes(cmd)?).map(|vec| {
   |                                                                ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::ListPushRequest`
   | 
  ::: src/lib.rs:59:8
   |
59 | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
   |        ----------           -------------- required by this bound in `protobytes`

error[E0599]: no function or associated item named `decode` found for type `wascc_codec::keyvalue::ListResponse` in the current scope
  --> src/kv.rs:88:38
   |
88 |             let resp = ListResponse::decode(vec.as_ref()).unwrap();
   |                                      ^^^^^^ function or associated item not found in `wascc_codec::keyvalue::ListResponse`
   |
   = help: items from traits can only be used if the trait is in scope
   = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
           `use prost::message::Message;`

error[E0277]: the trait bound `wascc_codec::keyvalue::ListDelItemRequest: prost::Message` is not satisfied
  --> src/kv.rs:98:68
   |
98 |         host_call(&route(CAPID_KEYVALUE, OP_LIST_DEL), &protobytes(cmd)?).map(|vec| {
   |                                                                    ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::ListDelItemRequest`
   | 
  ::: src/lib.rs:59:8
   |
59 | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
   |        ----------           -------------- required by this bound in `protobytes`

error[E0599]: no function or associated item named `decode` found for type `wascc_codec::keyvalue::ListResponse` in the current scope
  --> src/kv.rs:99:38
   |
99 |             let resp = ListResponse::decode(vec.as_ref()).unwrap();
   |                                      ^^^^^^ function or associated item not found in `wascc_codec::keyvalue::ListResponse`
   |
   = help: items from traits can only be used if the trait is in scope
   = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
           `use prost::message::Message;`

error[E0277]: the trait bound `wascc_codec::keyvalue::DelRequest: prost::Message` is not satisfied
   --> src/kv.rs:108:63
    |
108 |         host_call(&route(CAPID_KEYVALUE, OP_DEL), &protobytes(cmd)?).map(|_vec| ()).map_err(|e| e.into())
    |                                                               ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::DelRequest`
    | 
   ::: src/lib.rs:59:8
    |
59  | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
    |        ----------           -------------- required by this bound in `protobytes`

error[E0277]: the trait bound `wascc_codec::keyvalue::ListRangeRequest: prost::Message` is not satisfied
   --> src/kv.rs:117:65
    |
117 |         host_call(&route(CAPID_KEYVALUE, OP_RANGE), &protobytes(cmd)?).map(|vec| {
    |                                                                 ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::ListRangeRequest`
    | 
   ::: src/lib.rs:59:8
    |
59  | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
    |        ----------           -------------- required by this bound in `protobytes`

error[E0599]: no function or associated item named `decode` found for type `wascc_codec::keyvalue::ListRangeResponse` in the current scope
   --> src/kv.rs:118:43
    |
118 |             let resp = ListRangeResponse::decode(vec.as_ref()).unwrap();
    |                                           ^^^^^^ function or associated item not found in `wascc_codec::keyvalue::ListRangeResponse`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use prost::message::Message;`

error[E0277]: the trait bound `wascc_codec::keyvalue::ListClearRequest: prost::Message` is not satisfied
   --> src/kv.rs:127:65
    |
127 |         host_call(&route(CAPID_KEYVALUE, OP_CLEAR), &protobytes(cmd)?).map(|_vec| ()).map_err(|e| e.into())
    |                                                                 ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::ListClearRequest`
    | 
   ::: src/lib.rs:59:8
    |
59  | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
    |        ----------           -------------- required by this bound in `protobytes`

error[E0277]: the trait bound `wascc_codec::keyvalue::SetAddRequest: prost::Message` is not satisfied
   --> src/kv.rs:135:67
    |
135 |         host_call(&route(CAPID_KEYVALUE, OP_SET_ADD), &protobytes(cmd)?).map(|vec| {
    |                                                                   ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::SetAddRequest`
    | 
   ::: src/lib.rs:59:8
    |
59  | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
    |        ----------           -------------- required by this bound in `protobytes`

error[E0599]: no function or associated item named `decode` found for type `wascc_codec::keyvalue::SetOperationResponse` in the current scope
   --> src/kv.rs:136:46
    |
136 |             let resp = SetOperationResponse::decode(vec.as_ref()).unwrap();
    |                                              ^^^^^^ function or associated item not found in `wascc_codec::keyvalue::SetOperationResponse`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use prost::message::Message;`

error[E0277]: the trait bound `wascc_codec::keyvalue::SetRemoveRequest: prost::Message` is not satisfied
   --> src/kv.rs:146:70
    |
146 |         host_call(&route(CAPID_KEYVALUE, OP_SET_REMOVE), &protobytes(cmd)?).map(|vec| {
    |                                                                      ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::SetRemoveRequest`
    | 
   ::: src/lib.rs:59:8
    |
59  | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
    |        ----------           -------------- required by this bound in `protobytes`

error[E0599]: no function or associated item named `decode` found for type `wascc_codec::keyvalue::SetOperationResponse` in the current scope
   --> src/kv.rs:147:46
    |
147 |             let resp = SetOperationResponse::decode(vec.as_ref()).unwrap();
    |                                              ^^^^^^ function or associated item not found in `wascc_codec::keyvalue::SetOperationResponse`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use prost::message::Message;`

error[E0277]: the trait bound `wascc_codec::keyvalue::SetUnionRequest: prost::Message` is not satisfied
   --> src/kv.rs:154:69
    |
154 |         host_call(&route(CAPID_KEYVALUE, OP_SET_UNION), &protobytes(cmd)?).map(|vec| {
    |                                                                     ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::SetUnionRequest`
    | 
   ::: src/lib.rs:59:8
    |
59  | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
    |        ----------           -------------- required by this bound in `protobytes`

error[E0599]: no function or associated item named `decode` found for type `wascc_codec::keyvalue::SetQueryResponse` in the current scope
   --> src/kv.rs:155:42
    |
155 |             let resp = SetQueryResponse::decode(vec.as_ref()).unwrap();
    |                                          ^^^^^^ function or associated item not found in `wascc_codec::keyvalue::SetQueryResponse`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use prost::message::Message;`

error[E0277]: the trait bound `wascc_codec::keyvalue::SetIntersectionRequest: prost::Message` is not satisfied
   --> src/kv.rs:162:73
    |
162 |         host_call(&route(CAPID_KEYVALUE, OP_SET_INTERSECT), &protobytes(cmd)?).map(|vec| {
    |                                                                         ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::SetIntersectionRequest`
    | 
   ::: src/lib.rs:59:8
    |
59  | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
    |        ----------           -------------- required by this bound in `protobytes`

error[E0599]: no function or associated item named `decode` found for type `wascc_codec::keyvalue::SetQueryResponse` in the current scope
   --> src/kv.rs:163:42
    |
163 |             let resp = SetQueryResponse::decode(vec.as_ref()).unwrap();
    |                                          ^^^^^^ function or associated item not found in `wascc_codec::keyvalue::SetQueryResponse`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use prost::message::Message;`

error[E0277]: the trait bound `wascc_codec::keyvalue::SetQueryRequest: prost::Message` is not satisfied
   --> src/kv.rs:172:69
    |
172 |         host_call(&route(CAPID_KEYVALUE, OP_SET_QUERY), &protobytes(cmd)?).map(|vec| {
    |                                                                     ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::SetQueryRequest`
    | 
   ::: src/lib.rs:59:8
    |
59  | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
    |        ----------           -------------- required by this bound in `protobytes`

error[E0599]: no function or associated item named `decode` found for type `wascc_codec::keyvalue::SetQueryResponse` in the current scope
   --> src/kv.rs:173:42
    |
173 |             let resp = SetQueryResponse::decode(vec.as_ref()).unwrap();
    |                                          ^^^^^^ function or associated item not found in `wascc_codec::keyvalue::SetQueryResponse`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use prost::message::Message;`

error[E0277]: the trait bound `wascc_codec::keyvalue::KeyExistsQuery: prost::Message` is not satisfied
   --> src/kv.rs:182:70
    |
182 |         host_call(&route(CAPID_KEYVALUE, OP_KEY_EXISTS), &protobytes(cmd)?).map(|vec| {
    |                                                                      ^^^ the trait `prost::Message` is not implemented for `wascc_codec::keyvalue::KeyExistsQuery`
    | 
   ::: src/lib.rs:59:8
    |
59  | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
    |        ----------           -------------- required by this bound in `protobytes`

error[E0599]: no function or associated item named `decode` found for type `wascc_codec::keyvalue::GetResponse` in the current scope
   --> src/kv.rs:183:37
    |
183 |             let resp = GetResponse::decode(vec.as_ref()).unwrap();
    |                                     ^^^^^^ function or associated item not found in `wascc_codec::keyvalue::GetResponse`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use prost::message::Message;`

error[E0277]: the trait bound `wascc_codec::messaging::PublishMessage: prost::Message` is not satisfied
  --> src/msg.rs:61:25
   |
61 |             &protobytes(cmd)?,
   |                         ^^^ the trait `prost::Message` is not implemented for `wascc_codec::messaging::PublishMessage`
   | 
  ::: src/lib.rs:59:8
   |
59 | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
   |        ----------           -------------- required by this bound in `protobytes`

error[E0277]: the trait bound `wascc_codec::messaging::RequestMessage: prost::Message` is not satisfied
  --> src/msg.rs:77:25
   |
77 |             &protobytes(cmd)?,
   |                         ^^^ the trait `prost::Message` is not implemented for `wascc_codec::messaging::RequestMessage`
   | 
  ::: src/lib.rs:59:8
   |
59 | pub fn protobytes(msg: impl prost::Message) -> Result<Vec<u8>> {
   |        ----------           -------------- required by this bound in `protobytes`

warning: unused import: `prost::Message`
  --> src/kv.rs:25:5
   |
25 | use prost::Message;
   |     ^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

error: aborting due to 27 previous errors

Some errors have detailed explanations: E0277, E0599.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `wascc-actor`.

To learn more, run the command again with --verbose.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -15,7 +15,7 @@ categories = ["wasm", "api-bindings"]
 travis-ci = { repository = "wascc/wascc-actor", branch = "master" }
 
 [dependencies]
-wascc-codec = "0.3.4"
+wascc-codec = "0.3.5"
 wapc-guest = "0.1.1"
 prost = "0.6.1"
 serde_json = "1.0.46"

